.gitignore

```diff
@@ -1,3 +1,5 @@
+# vscode
```

Comment:
```suggestion
```

Reply:
Didn't we fix this (and some other changes) in a different PR? What happened here?

Reply:
Yeah, this PR should be merged after the other one. It has all commits from the other one. 

Since we changed things there, I need to keep rebasing this.